tests/testthat/_snaps/bump_version.md

```diff
@@ -0,0 +1,28 @@
+# bump_version() works
+
+    Code
+      bump_version()
+    Message <cliMessage>
+      > Scraping 3 commit messages.
+      v Found 1 NEWS-worthy entries.
+      
+      -- Updating NEWS --
+      
+      > Adding new entries to 'NEWS.md'.
+      
+      -- Update Version --
+      
+      v Package version bumped to 0.0.0.9001.
+      > Adding header to 'NEWS.md'.
+      > Committing changes.
+      
+      -- Tagging Version --
+      
+      > Creating tag v0.0.0.9001 with tag message derived from 'NEWS.md'.
+    Message <rlang_message>
```

Comment:
I'll add a remote on rlang for now https://github.com/r-lib/rlang/commit/300f932fddd4f3585913345c35dc22354600a1c6 -- before CRAN release the current CRAN rlang behavior should be checked. (or is this an argument for the fledge.quiet option + not snapshotting the messages? :sweat_smile: )

Reply:
@krlmlr any opinion on this? Is this PR good to merge?